tests/test_core.py

```diff
@@ -1411,6 +1431,8 @@ def test_exceptions_3(self):
     self.do_run(src, 'Caught exception: Hello\nDone.', ['2'], no_build=True)
 
   def test_exceptions_white_list(self):
+    if self.is_wasm_backend():
```

Comment:
Hmm, the decorator should make it not even run the contents of the function at all. Unless the decorator isn't working?